integration/spark/src/main/common/java/io/openlineage/spark/agent/lifecycle/OpenLineageRunEventBuilder.java

```diff
@@ -283,18 +299,18 @@ private RunEvent populateRun(
 
     RunFacetsBuilder runFacetsBuilder = openLineage.newRunFacetsBuilder();
     parentRunFacet.ifPresent(runFacetsBuilder::parent);
-    RunFacets runFacets = buildFacets(nodes, runFacetBuilders, runFacetsBuilder.build());
+    OpenLineage.JobFacets jobFacets =
+        buildFacets(nodes, jobFacetBuilders, openLineage.newJobFacets(null, null, null));
```

Comment:
Minor: Maybe we should define `newJobFacets()` with no params to avoid passing `null`s

Reply:
I could have used `newJobFacetsBuilder().build()`, but it would effectively accomplish the same thing.

Reply:
Cool. My thinking was around usability and avoiding `null`s, but either works I guess

Reply:
Sure, I think it would be a fine change to make in a separate PR 😀